stdlib/public/core/FixedPoint.swift.gyb

```diff
@@ -33,6 +33,21 @@ word_bits = int(CMAKE_SIZEOF_VOID_P) * 8
 # Number of bits in integer literals.
 builtinIntLiteralBits = 2048
 
+# Mapping from float bits to significand bits
+allFloatBits = [32, 64, 80]
+floatName = { 32:'Float', 64:'Double', 80:'Float80' }
+explicitFloatSignificandBits = { 32:24, 64:53, 80:64 }
```

Comment:
Could you push these constants into `SwiftIntTypes`, or a new module, `SwiftFloatingPointTypes`, so that we don't duplicate it?  Tests could also use it then.